### PR TITLE
Add support for symlinks

### PR DESCRIPTION
**Pull request by Blaimi**
Closes: #26

### CODE REVIEW

**Comment by Blaimi** [comment]
unittest is missing

File: index.js [hunk 1]
```diff
@@ -23,12 +23,16 @@ module.exports = (filename, options) => {
...
 		}
 
-		archive.append(file.contents, {
-			name: file.relative.replace(/\\/g, '/') + (file.isNull() ? '/' : ''),
-			mode: file.stat && file.stat.mode,
-			date: file.stat && file.stat.mtime ? file.stat.mtime : null,
-			...options
-		});
+		if (file.isSymbolic()) {
+			archive.symlink(file.relative.replace(/\\/g, '/'), file.symlink);
```
Comment (sindresorhus) [inline index.js]: `file.relative.replace(/\\/g, '/')` is repeated. Move it to a variable.

**Comment by sindresorhus** [comment]
This needs a test.

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**